scripts/lotus-provider.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Lotus Provider
+After=network.target
+After=lotus-daemon.service
+Wants=lotus-daemon.service
```

Comment:
Not sure if we want to require this, it reads as if we'd want lotus-deamon on the same machine, which in most production cases probably won't be the case